manifests/init.pp

```diff
@@ -116,6 +119,7 @@
   Integer[0] $keep_weekly                                  = 36,
   Integer[0] $keep_daily                                   = 60,
   Integer[0] $keep_within                                  = 30,
+  String[1] $compression                                   = 'lz4',
```

Comment:
should we turn this into an enum? there is only a small lost of allowed values

Reply:
No. One can provide different levels of compression like `zstd,10`.

Reply:
(Also there is stuff like `auto,zstd,5`)